cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -208,20 +244,33 @@ func generateMountsModeInstall(mst *initramfsMountsState) error {
 		return err
 	}
 
-	// 3. final step: write modeenv to tmpfs data dir and disable cloud-init in
-	//   install mode
-	modeEnv, err := mst.EphemeralModeenvForModel(model, snaps)
+	installAndRun, err := canInstallAndRunAtOnce(mst)
 	if err != nil {
 		return err
 	}
-	isRunMode := false
-	if err := modeEnv.WriteTo(boot.InitramfsWritableDir(model, isRunMode)); err != nil {
-		return err
-	}
 
-	// done, no output, no error indicates to initramfs we are done with
-	// mounting stuff
-	return nil
+	if installAndRun {
+		return fmt.Errorf("Not implemented")
```

Comment:
I know it will be implemented soon, but please add a TODO so when reading this all people is aware. Also, please add a more descriptive message not starting in capitals (like "install without reboot not implemented yet").

Reply:
Fixed as part of 5f3453adf2e88c4e9b9917f2bb9b054f67b67e80